src/lighteval/logging/info_loggers.py

```diff
@@ -86,9 +82,6 @@ class GeneralConfigLogger:
     model_dtype: str = None
     model_size: str = None
 
-    # Nanotron config
-    config: "Config" = None
```

Comment:
Why are we getting rid of the nanotron config logging ?

Reply:
I don't remember so either a merge issue or not something important, I can add it back

Reply:
yep would be great to keep it and remove it only if we are sure we don't need it

Reply:
reverted